### ISSUE TITLE
No module named input

### ISSUE THREAD

**Issue by Nofcale** [gunthercox/chatterbot-voice#5]
Hi gunther, when I'm trying to run the example.py file, I'm getting this error:

`InputError: No module named input`

But this is happening only when I use the VoiceInput and VoiceOutput in the input and output adapters, if I use any example of Chatterbot it runs fine so I'm not really sure what is happening.

Thanks.

Edit: I'm running this in a Raspberry Pi 3, it possible this issue is because it hasn't a micro plugged?

**Comment by gibbsie** [comment]
Same issue running the published example on Mac OS X 10.12.2 with chatterbot-voice-0.1.1, ChatterBot-0.4.13, SpeechRecognition-3.6.0:

```
Traceback (most recent call last):
  File "example.py", line 7, in <module>
    input_adapter="chatterbot_voice.VoiceInput",
...
ImportError: No module named input
```
And, if I comment out line 7:
```
Traceback (most recent call last):
  File "example.py", line 8, in <module>
    output_adapter="chatterbot_voice.VoiceOutput",
...
ImportError: No module named input
```
Thanks in advance!

**Comment by gunthercox** [comment]
Hi @Nofcale, @gibbsie, These modules were moved in the latest version of ChatterBot. I've updated the example. If you are interested in seeing what I changed to correct this, here is a link to the commit: https://github.com/gunthercox/chatterbot-voice/commit/2704238325184437675d9d9f7391f1b52355c9e7
